setup.py

```diff
@@ -38,7 +38,7 @@ def get_requirements(requirements_filename: str):
     return requirements
 
 
-PLUGIN_ENTRY_POINT = 'polyglot = neon_stt_plugin_polyglot:PolyglotSTT'
+PLUGIN_ENTRY_POINT = 'coqui = neon-stt-plugin-coqui:CoquiSTT'
```

Comment:
I think this should be `neon_stt_plugin_coqui:CoquiSTT` to match the module name

Reply:
Yes, that's it